src/components/HiveFactoryDashboard.module.css

```
@@ -2199,6 +2199,183 @@
+  width: 56px;
+  height: 56px;
+  border-radius: 50%;
+  border: 2px solid rgba(88, 166, 255, 0.35);
+}
+
+.spotlightName {
+  font-size: 0.78rem;
+  font-weight: 600;
+  color: #c9d1d9;
+  text-align: center;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)